torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -551,8 +551,8 @@ at::Tensor XLANativeFunctions::_to_copy(
 
   // Case 1: Materialize the tensor.
   if (device && device->type() != c10::kXLA) {
-    XLA_CHECK(device->type() == c10::kCPU)
-        << "only cpu device is supported in _to_copy.";
+    // XLA_CHECK(device->type() == c10::kCPU)
+    //     << "only cpu device is supported in _to_copy.";
     auto self_tensor = bridge::GetXlaTensor(self);
     auto eager_tensor = self_tensor->ToTensor(/*detached=*/true);
```

Comment:
Doesn't this code brings the tensor to CPU?

Reply:
Right. The eager_tensor after the line `auto eager_tensor = self_tensor->ToTensor(/*detached=*/true);` appear to be on CPU.

Do you need this PR to experiment the cuda->cpu->xla option? If not, I'll drop the PR and try to find a more direct way.

Reply:
Not really. Sorry for the late reply, by the way.